services/AuthService.js

```diff
@@ -120,33 +126,105 @@ module.exports = class AuthService {
 
     return ResponseMessage({
       statusCode: 401,
-      message: "Incorrect email and/or password.",
+      message: 'Incorrect email and/or password.',
       success: false
     })
   }
-  async verifyEmail(req) {
-    const confirmationToken = req.query.confirmationToken;
+
+  async verifyEmail (req) {
+    const confirmationToken = req.query.confirmationToken
 
     const user = await userModel.findOne({
       confirmationToken: confirmationToken
```

Comment:
Check also email
`
findOne({ email, confirmationToken })
`
If confirmation tolen is jwt, u can just verify 
` jwt.verify(..., ...)
`